src/3d/symbols/qgspolygon3dsymbol.h

```diff
@@ -186,9 +186,9 @@ class _3D_EXPORT QgsPolygon3DSymbol : public QgsAbstract3DSymbol SIP_NODEFAULTCT
 
   private:
     //! how to handle altitude of vector features
-    Qgis::AltitudeClamping mAltClamping = Qgis::AltitudeClamping::Relative;
+    Qgis::AltitudeClamping mAltClamping = Qgis::AltitudeClamping::Absolute;
     //! how to handle clamping of vertices of individual features
-    Qgis::AltitudeBinding mAltBinding = Qgis::AltitudeBinding::Centroid;
+    Qgis::AltitudeBinding mAltBinding = Qgis::AltitudeBinding::Vertex;
```

Comment:
Vertex mode doesn't work reliably for many polygons (same in 3d line symbols) -- can you drop this change and leave the relative -> absolute change only?

Reply:
I just pushed it corrected !

> Vertex mode doesn't work reliably for many polygons (same in 3d line symbols)
Have you an idea of what's messing with vertex binding ? (It seems like the correct way of binding "MultiPoint" geometries...)

Reply:
> Have you an idea of what's messing with vertex binding ? (It seems like the correct way of binding "MultiPoint" geometries...)

It works fine for multipoint. For polygons/linestrings it depends on the geometry itself -- but the issue is that the tesselator gives bad results for polygons geometries with varying z, if they don't form flat surfaces. And with buffered 3d lines the buffer process itself (from qgsline3dsymbol_p.cpp:157) strips the z coordinates.